soc/arm/actions/atb110x/Kconfig.defconfig.series

```diff
@@ -0,0 +1,34 @@
+# ATB110X series
+
+# Copyright (c) 2019, Manivannan Sadhasivam <mani@kernel.org>
+# SPDX-License-Identifier: Apache-2.0
+
+if SOC_SERIES_ATB110X
+
+config SOC_SERIES
+	default "atb110x"
+
+config NUM_IRQS
+	# must be >= the highest interrupt number used
+	default 32
+
+config ISR_STACK_SIZE
```

Comment:
Why does this need setting here?  probably a comment at min.

Reply:
The stack size is common for all of this SoC series